website/client/components/inventory/stable/petItem.vue

```diff
@@ -3,7 +3,7 @@ div
   .item-wrapper(@click="click()", :id="itemId")
     .item(
       :class="{'item-empty': emptyItem, 'highlight': highlightBorder}",
-    )
+    ).pet-slot
```

Comment:
`.pet-slot` should be after `.item` before the attributes

Reply:
it already is?

Reply:
can you give me a snippet for clarity please?

Reply:
sorry what I meant was `.item.pet-slot(`

Reply:
alrighty, I couldn't `git revert` the two files you requested, is there an easier way?

Reply:
also for the stuff under `.pet-slot` lines 7 I left them be.